src/lib/y2partitioner/actions/controllers/fstabs.rb

```diff
@@ -0,0 +1,237 @@
+        #
+        # @return [Array<Y2Storage::Fstab>]
+        def fstabs
+          disk_analyzer.fstabs
+        end
+
+        # Selects the previous fstab
+        #
+        # The current selected fstab does not change if it already is the first one.
+        #
+        # @return [Y2Storage::Fstab]
+        def select_prev_fstab
+          current_index = fstabs.index(selected_fstab)
+          prev_index = [0, current_index - 1].max
+
+          @selected_fstab = fstabs.at(prev_index)
+        end
+
+        # Selects the next fstab
+        #
+        # The current selected fstab does not change if it already is the last one.
+        #
+        # @return [Y2Storage::Fstab]
+        def select_next_fstab
+          current_index = fstabs.index(selected_fstab)
+          next_index = [fstabs.size - 1, current_index + 1].min
+
+          @selected_fstab = fstabs.at(next_index)
+        end
+
+        # Checks whether the selected fstab is the first one
+        #
+        # @return [Boolean]
+        def selected_first_fstab?
+          selected_fstab == fstabs.first
+        end
+
+        # Checks whether the selected fstab is the last one
+        #
+        # @return [Boolean]
+        def selected_last_fstab?
+          selected_fstab == fstabs.last
+        end
+
+        # Errors in the selected fstab
+        #
+        # @see #not_importable_entries_error
+        #
+        # @return [Array<String>]
+        def selected_fstab_errors
+          [not_importable_entries_error].compact
+        end
+
+        # Imports mount points from the selected fstab
+        #
+        # Before importing, the current devicegraph is reset to the system one.
+        def import_mount_points
+          reset_current_graph
+          importable_entries.each { |e| import_mount_point(e) }
+        end
+
+      private
+
+        # System devicegraph
+        #
+        # @return [Y2Storage::Devicegraph]
+        def system_graph
+          DeviceGraphs.instance.system
+        end
+
+        # Current devicegraph
+        #
+        # @return [Y2Storage::Devicegraph]
+        def current_graph
+          DeviceGraphs.instance.current
+        end
+
+        # Disk analyzer for the system devicegraph
+        #
+        # @return [Y2Storage::DiskAnalyzer]
+        def disk_analyzer
+          DeviceGraphs.instance.disk_analyzer
+        end
+
+        # Error when some entries in the selected fstab cannot be imported
+        #
+        # An entry cannot be imported when the device is not found or it is used
+        # by other device (e.g., used by LVM or MD RAID).
+        #
+        # @return [String, nil] nil if all entries can be imported
+        def not_importable_entries_error
+          entries = not_importable_entries
+          return nil if entries.empty?
+
+          mount_points = entries.map(&:mount_point).join("\n")
+
+          format(_("The following mount points cannot be imported:\n%{mount_points}"),
```

Comment:
Named placeholders like `%{mount_points}` are not very common outside the Ruby world, so better add a TRANSLATORS comment advising to not modify the content inside the `%{}`